stacks/lemp/service-config/nginx.conf

```diff
@@ -0,0 +1,82 @@
+worker_processes 4;
+pid /var/run/nginx.pid;
+
+events {
+	worker_connections 768;
+	# multi_accept on;
+}
+
+http {
+	# Basic Settings
+	sendfile on;
+	tcp_nopush on;
+	tcp_nodelay on;
+	keepalive_timeout 65;
+	types_hash_max_size 2048;
+	# server_names_hash_bucket_size 64;
```

Comment:
This commented-out config entry has been the source of many problems. Not a show-stopper, just curious if you understand http://nginx.org/en/docs/http/server_names.html yet. I don't yet.